docs/user/reuse_garage_policy.md

```diff
@@ -0,0 +1,163 @@
+# Reusing a policy trained with Garage
+
+In this section you will learn how to save and load snapshots of your experiment
+while training. We will also show you how to extract a trained policy from a
+snapshot.
+
+## Saving an experiment snapshot
```

Comment:
Just refer back to the experiment documentation for how to get one of these. No need to recapitulate it here. This doc can assume you already have run an experiment.